src/gui/topbar_subwindows/gui_military_window.hpp

```diff
@@ -42,8 +42,8 @@ class military_mob_progress_bar : public progress_bar {
 		int32_t used = 0;
 		int32_t total = 0;
 		for(const auto po : state.world.nation_get_province_ownership(n)) {
-			used += military::regiments_created_from_province(state, po.get_province());
-			total += military::regiments_max_possible_from_province(state, po.get_province());
+			used += military::mobilized_regiments_created_from_province(state, po.get_province());
+			total += military::mobilized_regiments_possible_from_province(state, po.get_province());
```

Comment:
I am afraid that this won't work either. There can be more mobilized regiments possible from the provinces collectively than the nation is allowed to raise, and thus the mobilization progress bar will never hit 100% with this change.